nixos/modules/services/networking/wireguard.nix

```diff
@@ -26,9 +26,10 @@ let
         type = with types; nullOr str;
         default = null;
         description = ''
-          DEPRECATED BECAUSE IT PUTS A WORLD-READABLE COPY OF THE KEY IN /nix/store!
-
           Base64 private key generated by wg genkey.
+
+          WARNING: Consider using privateKeyFile instead if you don't
```

Comment:
I'm not much of a fun of this warning.

If you really insist on having a warning like this, don't use all caps, and say "do not" instead of "don't."

But meh.